client/app/components/service-status/service-status-action.component.js

```diff
@@ -0,0 +1,15 @@
+(() => {
+  angular.module('directives').component('serviceStatusAction', {
+    templateUrl: 'components/service-status/service-status-action.component.html',
```

Comment:
You can `import templateUrl from './components/service-status/service-status-action.component.html''`